Software/spartan/src/sensors/mtk3339.cpp

```diff
@@ -0,0 +1,28 @@
+#include "mtk3339.h"
+
+// Constructor
+
+spartan::MTK3339::MTK3339(int bus, uint8_t address)
+    : Sensor(bus, address), m_i2c(bus, true)
+{}
+
+// Identification
+
+const char* spartan::MTK3339::name() const 
+{
+    const char * str = "MTK3339";
+    return str;
+};
+
+void spartan::MTK3339::printSensorInfo() {}
+
+
+// Power functions
+
+int spartan::MTK3339::powerOn() { return 0; }
+int spartan::MTK3339::powerOff() { return 0; }
+
+// Update sensor data buffer
+int spartan::MTK3339::poll(MasterDataPacket &dp) { return 0; }
```

Comment:
Recently there was some changes to the `Sensor` base class interface implemented in #49... Before merging, we should make sure this skeleton also obeys this update.